src/service.ts

```diff
@@ -163,6 +167,12 @@ export default class VSCodeWorkerService implements Services.ServiceInstance {
         log.info(`Start VSCode: ${binary} ${args.join(' ')}`)
     }
 
+    beforeCommand (commandName: string): void {
+        if (commandName === 'deleteSession') {
```

Comment:
Let's also verify that the session is Cucumber and add a comment documenting why we have this special case for it.

Reply:
**Done**. It is set in `beforeSession` to have access to `config`.